### ISSUE TITLE
Release version 13 on pursuit

### ISSUE THREAD

**Issue by roryc89** [alexmingoia/purescript-pux#159]
Although version 13 can be used by referencing the git repo it would be nice to have new version and documentation on pursuit

**Comment by nukisman** [comment]
@alexmingoia what is the current problem with publication docs on pursuit?

**Comment by MichaelXavier** [comment]
Just gently bumping this. It is a bit tricky to work on pux projects without the docs being published.

**Comment by alexmingoia** [comment]
I haven't been able to publish because `pulp publish` blows the stack and crashes. Giving it more memory doesn't seem to help. It might be related to purescript-contrib/pulp#351 

I'm not sure how to proceed. 

```
λ pux (master): node --max_old_space_size=10000 /usr/local/lib/node_modules/pulp/pulp.js publish
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
 1: 0x100035d27 node::Abort() [/usr/local/bin/node]
 2: 0x1000374a6 node::FatalTryCatch::~FatalTryCatch() [/usr/local/bin/node]
 3: 0x10018397f v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/usr/local/bin/node]
 4: 0x100183921 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/usr/local/bin/node]
 5: 0x100475f78 v8::internal::Heap::UpdateSurvivalStatistics(int) [/usr/local/bin/node]
 6: 0x10047c5d9 v8::internal::Heap::AllocateRawWithRetry(int, v8::internal::AllocationSpace, v8::internal::AllocationAlignment) [/usr/local/bin/node]
 7: 0x10045a9f1 v8::internal::Factory::NewRawTwoByteString(int, v8::internal::PretenureFlag) [/usr/local/bin/node]
 8: 0x100544b13 v8::internal::String::SlowFlatten(v8::internal::Handle<v8::internal::ConsString>, v8::internal::PretenureFlag) [/usr/local/bin/node]
 9: 0x1005f06d7 v8::internal::RegExpImpl::IrregexpExec(v8::internal::Handle<v8::internal::JSRegExp>, v8::internal::Handle<v8::internal::String>, int, v8::internal::Handle<v8::internal::RegExpMatchInfo>) [/usr/local/bin/node]
10: 0x100680dd7 v8::internal::Runtime_RegExpExec(int, v8::internal::Object**, v8::internal::Isolate*) [/usr/local/bin/node]
11: 0x29ceaf6841bd

<--- Last few GCs --->

[2094:0x104004c00]     6415 ms: Mark-sweep 1142.4 (1253.3) -> 1142.3 (1219.3) MB, 89.9 / 0.0 ms  (average mu = 0.730, current mu = 0.000) last resort GC in old space requested
[2094:0x104004c00]     6494 ms: Mark-sweep 1142.3 (1219.3) -> 1142.3 (1219.3) MB, 78.5 / 0.0 ms  (average mu = 0.564, current mu = 0.000) last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 0x29ceaf6841bd]
    1: StubFrame [pc: 0x29ceaf6cec3b]
Security context: 0x353cfcc1e6c9 <JSObject>
    2: replace [0x353cfcc10d11](this=0x353c8562a8b1 <Very long string[566411532]>,0x353c87397a51 <JSRegExp <String[74]: [\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-PRZcf-nqry=><]>>,0x353c44082441 <String[0]: >)
    3: stripColor [0x353c8de63be9] [/usr/local/lib/node_modules/bower/lib/node_modules/strip-an...

* ERROR: Subcommand terminated as a result of receiving SIGABRT
```

**Comment by hdgarrood** [comment]
I've just published v13.0.0 for you (using an unreleased version of the compiler). The fix for this issue on the compiler side has been merged and will be in the next release, and the fix for Pulp is in https://github.com/purescript-contrib/pulp/pull/370, so hopefully this should just work soon.

**Comment by alexmingoia** [comment]
Thanks!
